package.json

```diff
@@ -21,20 +24,59 @@
   },
   "homepage": "https://github.com/cachilders/backpat#readme",
   "devDependencies": {
+    "babel-cli": "^6.18.0",
+    "babel-plugin-istanbul": "^3.0.0",
+    "babel-preset-es2015": "^6.18.0",
+    "babel-register": "^6.18.0",
     "chai": "^3.5.0",
     "chai-spies": "^0.7.1",
     "coveralls": "^2.11.15",
+    "cross-env": "^3.1.3",
     "eslint": "^3.8.1",
     "eslint-plugin-flowtype": "^2.25.0",
+    "ghooks": "^1.3.2",
```

Comment:
Okay, this is one of the coolest things I've learned about recently. Every time you do something in git (e.g. commit, push, pull, rebase, etc.) git first checks that command's hooks (check .git/hooks for more). You can write a CLI command to do something every time you run that git command. The problem with this, however, is .git folders are not committed to source control so these have been historically difficult to share. 
_Cue githooks._ 
See more below.